web/src/routes/(user)/photos/+page.svelte

```diff
@@ -31,6 +31,7 @@
   const { isMultiSelectState, selectedAssets } = assetInteractionStore;
 
   $: isAllFavorite = [...$selectedAssets].every((asset) => asset.isFavorite);
+  $: isAssetStackSelected = !!($selectedAssets.size === 1 && [...$selectedAssets][0].stack);
```

Comment:
Why do you copy the selectedAssets array first and then access the first element?

Reply:
Because it's a Set, not an Array

Reply:
Right, if was just asking whether getting the iterator with `$selectedAssets.values()` is more performant.

Reply:
I'm pretty sure the difference very negligible. What could be done however is having only one copy to compute both `isAllFavorite` and `isAssetStackSelected`

Reply:
Ha nice! Thank you :)